blosc/blosc2.c

```diff
@@ -19,8 +19,8 @@
 
 #include "blosc2.h"
 #include "blosc-private.h"
-#include "blosc2-common.h"
-#include "blosc2-stdio.h"
+#include "blosc2/blosc2-common.h"
+#include "blosc2/blosc2-stdio.h"
```

Comment:
This two includes should be included inside the #if defined(HAVE_PLUGINS) condition.